src/query/procedure/callable_alias_mapper.hpp

```diff
@@ -32,10 +32,10 @@ class CallableAliasMapper final {
   ~CallableAliasMapper() = default;
 
   void LoadMapping(const std::filesystem::path &);
-  [[nodiscard]] std::optional<std::string_view> FindAlias(const std::string &) const noexcept;
+  [[nodiscard]] std::optional<std::string_view> FindAlias(std::string_view name) const noexcept;
 
  private:
-  std::unordered_map<std::string, std::string> mapping_;
+  std::map<std::string, std::string, std::less<>> mapping_;
```

Comment:
To avoid needing to allocate `std::string` for lookups. I prefer to pass around `std::string_view`. We could use a heterogeneous hash function for strings if needed, but for now I am happy to move from unordered_map to map, until a performance related issue highlights that it needs to be a unordered_map.